test/report.js

```diff
@@ -0,0 +1,140 @@
+/* global describe, it */
+const { MockReport } = require('./report-mock.js')
+const { buildYargs, hideInstrumenteeArgs } = require('../lib/parse-args')
+const { existsSync, mkdirSync } = require('fs')
+const { resolve } = require('path')
+const chai = require('chai')
+const spies = require('chai-spies')
+
+chai.use(spies)
+const { expect } = chai
+const nodePath = process.execPath
+const nodeMajorVersion = Number(process.version.slice(1).split('.')[0])
+
+describe('report', () => {
+  if (nodeMajorVersion > 10) {
+    /**
+     * cause Report._loadReports to throw an error and catch it to increase code coverage
+     *
+     * Strategy:
+     * Add a non json file to the temporary directory to cause _loadReports to throw
+     * an error.  Do this before running reports constructor and after manually creating
+     * the temporary directory
+     *
+     */
+    it('cause Report._loadReports to throw an error and catch it', async () => {
```

Comment:
> These are private methods, rather than forcing them to throw, it would be better if we could figure out a condition that caused loadReports to throw an exception, when invoked through the public interface.

Conditions that would cause the exception to be thrown will be discovered over time through bug reports.

From my perspective, it can be difficult to tell what application state can cause bugs.  That is why they are bugs.  It's an application state that you didn't anticipate, and there is no logic written for that application state.  In certain cases, it may be an invalid application state or at other times it could be a feature not working as intended. It's important to test that exceptions are thrown and working as expected, then as bugs arise, hopefully, it's covered by an exception.